src/main/java/org/embulk/output/s3/S3FileOutputPlugin.java

```diff
@@ -116,6 +116,11 @@ public interface PluginTask
         @Config("region")
         @ConfigDefault("null")
         Optional<String> getRegion();
+
+        @Config("path_style_access")
+        @ConfigDefault("false")
+        Optional<Boolean> getPathStyleAccess();
+
```

Comment:
If it has `ConfigDefault`, it does not need to be `Optional` unless it can be `null`.

```suggestion
        @Config("use_path_style_access")
        @ConfigDefault("false")
        boolean getPathStyleAccess();
```